### PR TITLE
Completed react-state exercise. - Charlie McCanna

### PR DESCRIPTION
**Pull request by cgmccanna**
react-state

### CODE REVIEW

File: react-state/my-app/src/ToggleButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+export default function ToggleButton({text, color}) {
+  const [isClicked, setClicked] = useState(false);
```
Comment (dseongdev) [inline react-state/my-app/src/ToggleButton.js]: the setter should be named 'setIsClicked'. Not a big deal but just good practice to follow that convention.

File: react-state/my-app/src/ToggleButton.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  console.log('useState:', isClicked)
+
+  function handleClick() {
+    console.log('before calling setter:', isClicked)
+    setClicked(!isClicked);
+    console.log('after calling setter:', isClicked)
+  }
+
+  if (isClicked) {
```
Comment (dseongdev) [inline react-state/my-app/src/ToggleButton.js]: Does this fulfill instruction #7?

**Review by dseongdev — CHANGES_REQUESTED** [review]
just a couple things. almost there.

**Comment by cgmccanna** [comment]
react-state-2

File: react-state/my-app/src/ToggleButton.js (hunk 2 above)
Comment (dseongdev) [reply react-state/my-app/src/ToggleButton.js]: The code you had before was fine. 
#7 of the instructions want you to set the buttons to white by default. 
so the change should be made in the conditional.

**Review by dseongdev — CHANGES_REQUESTED** [review]

**Comment by cgmccanna** [comment]
react-state-3

**Review by dseongdev — APPROVED** [review]

**Pull request merged**